server/app/services/export/enums/SubmitterType.java

```diff
@@ -0,0 +1,6 @@
+package services.export.enums;
+
+public enum SubmitterType {
```

Comment:
**suggestion (blocking):** add javadoc for the enum

Reply:
Done